ci-operator/step-registry/ibmcloud/provision/vpc/ibmcloud-provision-vpc-commands.sh

```diff
@@ -231,12 +272,17 @@ if [[ "${APPLY_ALL_SUBNETS}" == "no" ]]; then
         compute_idx=$(( RANDOM % compute_subnets_len))
       ;;
       *)
-        echo "unsupported value for PICKUP_SUBNETS_IN_ORDER"
+        echo "unsupported value for PICKUP_SUBNETS_ORDER"
         exit 2
       esac
       cat "${vpc_info_file}" | jq -c -r --arg z "${zone}" --argjson idx "$cp_idx" '[[.subnets | sort_by(.created_at) | .[] | select(.zone.name==$z) | select(.name|test("control-plane")) | .name][$idx]]' | yq-go r -P - >>${workdir}/controlPlaneSubnets.yaml
       cat "${vpc_info_file}" | jq -c -r --arg z "${zone}" --argjson idx "$compute_idx" '[[.subnets | sort_by(.created_at) | .[] | select(.zone.name==$z) | select(.name|test("compute")) | .name][$idx]]' | yq-go r -P - >>${workdir}/computerSubnets.yaml
     done
+
+    if  [[ $PICKUP_SUBNETS_ORDER == "ascending" ]]; then
+        checkUsedSubnets "${workdir}/controlPlaneSubnets.yaml" "${resource_group}" || exit 1
```

Comment:
If you want to accurately pick up the subnets in the 2nd page, you should introduce a new option of `PICKUP_SUBNETS_ORDER` for that.